Document QGIS Development Server standalone application (#5309)
qgis/QGIS-Documentation: docs/user_manual/working_with_ogc/server/development_server.rst

.. index::
    pair: Development; QGIS Server

.. _server_development_server:

**********************
Development Server
**********************

A production installation and deployment of QGIS Server usually involves setting
up a web server component (e.g. Apache or Nginx) that can forward the HTTP requests
coming from the clients to the QGIS Server FastCGI binary application.

If you want to quickly test QGIS Server on your local machine without configuring

> and installing a full web server stack you can use the new QGIS Development Standalone

Reviewer: ```suggestion
and installing a full web server stack you can use the QGIS Development Standalone
```
I tend to consider these time-related information more suitable for changelog. When 3.16 doc is released, this will no longer be a new feature and we'll probably not reword this so may be better to skip it right now?

server.

This is an independent application that provides a very simple web server ready
to serve your project files.

.. warning::

    The Standalone Development Server has not been developed with the purpose
    of being used in production, it was not checked for security vulnerabilities

>     or for other stress conditions that usually happen on a publicly exposed

Reviewer 2: ```suggestion
    or for other stress conditions that normally will occur on a publicly exposed
```

    server.

To launch the server:

.. code-block:: bash

    $ qgis_mapserver

> The default port the Development Server listens to is ``8000``, example output:

Reviewer 2: ```suggestion
The default port the Development Server listens to is ``8000``. Example output:
```

.. code-block:: bash

    QGIS Development Server listening on http://localhost:8000
    CTRL+C to exit
    127.0.0.1 [lun gen 20 15:16:41 2020] 5140 103ms "GET /wfs3/?MAP=/tests/testdata/qgis_server/test_project.qgs HTTP/1.1" 200
    127.0.0.1 [lun gen 20 15:16:41 2020] 3298 2ms "GET /wfs3/static/jsonFormatter.min.js HTTP/1.1" 200
    127.0.0.1 [lun gen 20 15:16:41 2020] 1678 3ms "GET /wfs3/static/jsonFormatter.min.css HTTP/1.1" 200
    127.0.0.1 [lun gen 20 15:16:41 2020] 1310 5ms "GET /wfs3/static/style.css HTTP/1.1" 200
    127.0.0.1 [lun gen 20 15:16:43 2020] 4285 13ms "GET /wfs3/collections?MAP=/tests/testdata/qgis_server/test_project.qgs HTTP/1.1" 200

> The server has a few options that can be passed as command line arguments, you can see them all

Reviewer 2: ```suggestion
The server has a few options that can be passed as command line arguments.
You can see them all by invoking the server with ``-h``.
```

> by invoking the server with ``-h``.

Reviewer 2: ```suggestion
```
